definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -179,12 +182,18 @@ declare module "lodash" {
     | matchesPropertyIterateeShorthand
     | propertyIterateeShorthand;
 
+  declare type IterateeWithResult<V, O, R> =
+    | ((value: V, key: string, object: O) => R)
+    | string;
+
   declare type OIterateeWithResult<V, O, R> =
-    | Object
-    | string
-    | ((value: V, key: string, object: O) => R);
+    | ReadOnlyIndexerObject<V>
```

Comment:
`Object` is `any` in Flow, unfortunately. Therefore, it was replaced with more precise type